hack/manifests/receivers/dynamic/main.go

```
@@ -0,0 +1,233 @@
+	}
+}
+
+func (s *Server) Stop() {
+	fmt.Println("[SERVER] Initiating silent black-hole teardown...")
+
+	close(s.stopChan)
+
+	s.mu.Lock()
+	for _, conn := range s.activeClients {
+		if tcpConn, ok := conn.(*net.TCPConn); ok {
+			tcpConn.SetLinger(0)
```

golangci-lint (2.13.2)

[error] Error return value of `tcpConn.SetLinger` is not checked

(errcheck)